Cython/Compiler/Nodes.py

```diff
@@ -5081,22 +5081,27 @@ def generate_type_ready_code(entry, code, heap_type_bases=False):
             tuple_temp = code.funcstate.allocate_temp(py_object_type, manage_ref=True)
             base_type = scope.parent_type.base_type
             if base_type:
-                for slot in TypeSlots.slot_table:
-                    if slot.slot_name == "tp_base":
-                        code.putln(
-                            "%s = PyTuple_Pack(1, (PyObject *)%s); %s" % (
-                            tuple_temp,
-                            base_type.typeptr_cname,
-                            code.error_goto_if_null(tuple_temp, entry.pos)))
-                        code.put_gotref(tuple_temp)
-            code.putln(
-                "%s = PyType_FromSpecWithBases(&%s_spec, %s); %s" % (
-                    typeobj_cname,
-                    typeobj_cname,
+                slot = TypeSlots.get_slot_by_name("tp_base")
```

Comment:
Erm … where is this used now?

Reply:
Oops, it's not used. I was using it to find if a type had a `tp_base` but the `if base_type` check is good enough so this can be removed.